### DISCUSSION TITLE
How to add M2M rows that are related via ORM Model

### DISCUSSION THREAD

**Discussion by anna-osipchyk** [aminalaee/sqladmin#446] [Q&A] [answered]
hello! I have 3 models:

  class GameQuest(Base):
      __tablename__ = "game_quest"
      __tableargs__ = UniqueConstraint("game_id", "quest_id")
  
      id = Column(Integer, primary_key=True)
      game_id = Column(Integer, ForeignKey("game.id"), nullable=False)
      quest_id = Column(Integer, ForeignKey("quest.id"), nullable=False)
      quest = relationship("Quest", back_populates="game_association", uselist=False)
      game = relationship("Game", back_populates="quest_association", uselist=False)
      winner = relationship("Winner", back_populates="game_quest", uselist=False, cascade="all, delete, delete-orphan")

  class Quest(Base):
      __tablename__ = "quest"

      id = Column(Integer, primary_key=True)
      name = Column(String(100), nullable=False, unique=True)
      type = Column(String, nullable=False)
      game_association = relationship("GameQuest", back_populates="quest")
      games = association_proxy("game_association", "game")
  
  class Game(Base):
      __tablename__ = "game"

      id = Column(Integer, primary_key=True)
      name = Column(String(100), nullable=False, unique=True)
      is_active = Column(Boolean, default=True)
      quest_association = relationship("GameQuest", back_populates="game")
      quests = association_proxy("quest_association", "quest")
    

How can i properly add new Games while creating Quest?
For now i can do it only via GameQuest:
![image](https://user-images.githubusercontent.com/82057714/224744276-dc253e4f-2252-4197-a8bf-bdb71d87fd52.png)

Thank you in advance!

**Comment by aminalaee** [comment, marked as answer]
I think right now this is the only option.
Usually this should be solved with InlineModel to create both Game and GameQuest in one page.
There's already a feature request for this.

**Reply by anna-osipchyk** [reply]
Okay, thank you!
